lib/signs_ui_web/templates/unauthorized/index.html.eex

```diff
@@ -0,0 +1,3 @@
+You are not authorized to access the signs viewer.
+If you believe you should have access, please have your supervisor contact
+<a href=\"realtimeappsdl@mbta.com\">realtimeappsdl@mbta.com</a>.
```

Comment:
The backslashes are superfluous now. Also, I don't think this should be a link anyway. Maybe a `mailto:` but actually probably not since this person isn't the one doing the mailing.

Reply:
Fixed.